masonry/src/widgets/button.rs

```diff
@@ -226,12 +237,19 @@ impl Widget for Button {
         let bg_rect = border_width.bg_rect(size, border_radius);
         let border_rect = border_width.border_rect(size, border_radius);
 
-        let border_color = if is_hovered && !ctx.is_disabled() {
+        let mut border_color = if is_hovered && !ctx.is_disabled() {
             &props.get::<HoveredBorderColor>().0
         } else {
             props.get::<BorderColor>()
         };
 
+        // FIXME - Handle this properly
+        if ctx.is_focus_target() {
+            border_color = &BorderColor {
+                color: Color::WHITE,
+            };
+        }
```

Comment:
Drawing the focus outline should be done centrally in the `paint` pass. Not inside of the widget.

Reply:
Disagree. I think it should ultimately be stylable, once we add pseudo-classes to the styling system.

Reply:
That's a valid point. I'd recommend opening an issue about it, because I don't think this needs to be resolved in this PR.

Reply:
I had implemented this locally in 10 lines without noticing any complications (might not be perfect but still better than nothing) ... but sure I can make a PR for that.